scripts/install.sh

```diff
@@ -22,21 +22,7 @@ fi
 
 if command_exists socketplane; then
     echo >&2 'Warning: "socketplane" command appears to already exist.'
-    while true; do
-        read -p "Would you like to re-install socketplane (y/n) " yn
-        case $yn in
-            [Yy]* )
-                cleanup
-                break
-                ;;
-            [Nn]* )
-                exit 1
-                ;;
-            * )
-                echo "Please answer yes or no."
-                ;;
-        esac
-    done
+    echo >&2 'Warning: "Please run "socketplane uninstall first'
```

Comment:
Oh, and I didn't see an `exit 1` so I assume execution continues.
May be worth adding a `sleep 20` to allow the user time to CTRL+C if they really didn't mean to re-install